### DISCUSSION TITLE
2.1.754 New Model List

### DISCUSSION THREAD

**Discussion by lllyasviel** [lllyasviel/Fooocus#808] [Announcements]
The default model list is updated in 2.1.754:

| Task | Windows | Linux args | Main Model | Refiner | Config |
| - | - | - | - | - | - |
| General | run.bat |  | [juggernautXLv6_RunDiffusion](https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/juggernautXL_version6Rundiffusion.safetensors) | not used | [here](https://github.com/lllyasviel/Fooocus/blob/main/modules/path.py) |
| Realistic | run_realistic.bat | --preset realistic | [realistic_stock_photo](https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/realisticStockPhoto_v10.safetensors) | not used | [here](https://github.com/lllyasviel/Fooocus/blob/main/presets/realistic.json) |
| Anime | run_anime.bat | --preset anime | [bluepencil_v50](https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/bluePencilXL_v050.safetensors) | [dreamsharper_v8](https://huggingface.co/lllyasviel/fav_models/resolve/main/fav/DreamShaper_8_pruned.safetensors) (SD1.5) | [here](https://github.com/lllyasviel/Fooocus/blob/main/presets/anime.json) |

**The change will lead to these results:**

(1) If you do nothing, nothing will change. Your model will not be changed. No download will be forced. This is because we try best to avoid forcing users to download new big files, so as to keep Fooocus user-friendly.

(2) OR, if you want to try what state-of-the-art is like, and what people are achieving in recent community, you can 

* delete `Fooocus\user_path_config.txt` and your Fooocus will be refreshed. Newer models will be downloaded in the next time you use `run.bat` or `run_X.bat`, if newer models are not there. Remember to backup files if you want. (recommended)
* OR, download Fooocus again. (slow and heavy, but will always work)

**Comment by lllyasviel** [comment]
# 2.1.754 "run.bat" non-cherry-picked test

You need to delete `Fooocus\user_path_config.txt` and refresh default config (or just download Fooocus again). After you get juggernautXL v6 (it will be automatically downloaded if you delete  `user_path_config.txt`), you will be able to reproduce these:

### woman (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/ea673088-38d8-4ada-a621-8d84763ff518)

### a handsome man (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/9badcc55-7136-4ebf-be57-70111d2a66da)

### modern sports car (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/cd403abd-0978-42b4-b8ba-174eca1ffeb5)

### dog in front of a house (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/e3af3395-d10f-4250-a823-67d259e82b54)


### house in garden (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/b5645c72-67b2-42a3-8581-ff745c292636)

### Socrates tells a story (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/05b10bbe-a422-4c94-bd12-6fa343c6a253)

### fire fighting (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/5f82fd3e-5cac-4566-b904-ceddc30f3100)

### evil magician destroy the world (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/a321660f-d32a-406c-94e8-355683f5d714)

### the furry elves of the forest (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/6252fc2e-bc4a-444d-acd1-1462f692b90c)

### beautiful girl (Seed 31337, 16 non-cherry-picked images in a same batch)
![image](https://github.com/lllyasviel/Fooocus/assets/19834515/3aed46d0-8454-4cb5-841e-c9e99f30480d)

**Comment by 3Diva** [comment]
These look absolutely amazing! I'm definitely going to have to see about deleting that user_path_config.txt file and restarting. I'm looking forward to checking out this update! 

Edit: LOLing at the random giant hamsters in that one "house in garden" image and the random fashionista in Paris in the "evil magician destroy the world" image. HAHAH 

Kidding aside - these are outstanding images! Wow!

**Comment by 3Diva** [comment]
After doing some testing with it - I'm really enjoying the new settings! It creates really lovely images! Here are a few using some of the wildcard prompts from a set of romance wildcards I'm working on.

![2023-10-29_00-24-35_4607](https://github.com/lllyasviel/Fooocus/assets/51475772/cfbe3c7e-472e-45ae-84ed-2408fe7fd0b6)
![2023-10-29_00-26-54_7086](https://github.com/lllyasviel/Fooocus/assets/51475772/710debed-8afa-4c0d-bc0b-8544b4db1dd4)
![2023-10-29_00-34-13_7961](https://github.com/lllyasviel/Fooocus/assets/51475772/216879eb-31e7-4596-a5c3-ed17dc1528f3)

And here are a couple to celebrate the upcoming holiday:

![2023-10-29_00-40-38_7117](https://github.com/lllyasviel/Fooocus/assets/51475772/7ca4adcd-1c70-4458-8805-853d3fce5a2a)
![2023-10-29_01-14-18_3074](https://github.com/lllyasviel/Fooocus/assets/51475772/fcbf5d8f-a37c-4845-bf5d-3c329881d0c8)


I'm really liking the settings! Thank you!

**Reply by mirek190** [reply]
how to use wildcards?

**Reply by 3Diva** [reply]
@mirek190 Just make sure the wildcards are in the wildcards folder: Fooocus >> wildcards, and then copy the name of the wildcard and put two underscores in front of the wild card name and two underscores behind the wildcard name. For instance if you have a wildcard named "clothing" you would type this into the prompt area: `__clothing__`

**Reply by rundiffusion** [reply]
These are absolutely amazing!

**Comment by rundiffusion** [comment]
@lllyasviel 
Hi! RunDiffusion here! So excited to see Juggernaut+RD being added to the official model download list. We brought photo realism to Juggernaut though merging our WIP photo model. (See it here https://RunDiffusion.com/rundiffusion-photo) The agreement from Kandoo aka Juggernaut was to make sure the model was titled with our mix. Can we get some recognition for our part in Juggernaut v6 please? It’s 35% RunDiffusion.
Credit where credit is due right? Please make sure that the name of the model is downloaded and includes the name from Civitai. 

We would love a “Cloud Provider” mention in this repo as well. We’ve been pushing Fooocus to our many customers and we feel like we are helping the adoption.

We’re a self funding group of 5. If we are to continue our research. We need help and partners.

We’re a company trying to self finance research and open source. Our contributions include: 
Lots of Code submitted to Deforum to get it working on Linux. 
The builtin LDSR and ScuNET extensions for Auto are (mostly) our code. 
Helped fix a model scanning script in Invoke. 
The run button on Civitai was our idea with them.
Iframe issues in Auto 
Dozens of auto1111 extensions have pull requests from us 
We are actively working with the teams at SD.Next, Enfugue, Makeayo, Arthemy, SDXF. 
Patched filebrowser.org 
We have helped the team at EbSynth

We’re released 3 open source models including one of the worlds first SDXL models that have been used in hundreds of merges and other projects.

Worked closely with these model creators giving them free GPU resources. 
Juggernaut 
Joachim 
DeformerToby 
Triple Headed Monkey 
Crystal Clear team 
Team Synergix 
Dozens of smaller creators

We do free education live streams. Etc every week multiple times in our discord.

We love open source.

Please consider helping us get exposed for all the work we’re doing in the space. 

Please reach out privately if you’re we need to talk. We’re dedicated to making SD the best it can be. And we mean it.

**Reply by lllyasviel** [reply]
thanks i modified the title - see if it works

**Reply by lllyasviel** [reply]
@rundiffusion 
Besides, the choice of this list is based on non-cherry-picked user study. We need to 100% avoid any commercial influence on this list, and the list may change in the future if any models outperform any existing models in our tests. Nevertheless, feel free to notify us about any license details.

Thank you for the understanding the support to project Fooocus!

**Reply by rundiffusion** [reply]
Thanks so much! I see in the code that the name will be used. Which is great. 

We are self funded, I understand that you need to avoid affiliation. We’re just trying to fund the team to continue working. 

This is great so thank you. No license needed, the license is provided by Kandoo and Civitai is free and open.

We absolutely love Fooocus and will always support you guys. We would submit pull requests but it’s been rock solid! So no news is good news!

**Reply by rundiffusion** [reply]
@lllyasviel is there a better channel to communicate with you and your team? We would like to contribute more to your project.

**Reply by lllyasviel** [reply]
we do not use discord or any channel to avoid the “information bubble“ of only english-speaking users or any specific minority user group with louder voice. 

We monitor international feedbacks in all communities and all languages and all countries. 

For this repo, we only use github for everything.

**Reply by rundiffusion** [reply]
Understood! We will communicate here. Thanks for all your hard work on Fooocus. What an amazing tool you've built.

**Comment by KandooAI** [comment]
Wow, what an honor for me :)
Great pictures. The Shrek evil magician made me smile; he came out of nowhere :D

**Reply by 3Diva** [reply]
@KandooAI Are you one of the creators of the Juggernaut v6 along with @rundiffusion ? If so - excellent work! I'm really enjoying the checkpoint model you guys have created! I've tried many SDXL models and I'm using yours and CrystalClear XL the most as the results I get with those two models are just so good - Chef's Kiss! Thank you and RunDiffusion for all you guys' hard work and for creating a great checkpoint model! It's very much appreciated! A very cool team-up that's produced really nice results! 👍 ❤️

**Reply by rubenmejiac** [reply]
Would be awesome to have the new [Juggernaut XL v7 + RunDiffusion](https://civitai.com/models/133005/juggernaut-xl?modelVersionId=240840) upgrade the already wonderful v6 model as the default for Fooocus.

**Reply by kwilliams1987** [reply]
Evil magician Shrek caught me off guard.

**Comment by Imadbein46** [comment]
with great Fooocus comes great Images hh Thx
![Fooocus 2 1 761 - Google Chrome 10_30_2023 1_18_01 PM](https://github.com/lllyasviel/Fooocus/assets/56880122/fe9775cf-7c9f-48de-b59a-44084ab264d0)

**Reply by simartem** [reply]
can you provide model and style info, or preferably a screenshot from the log. amazing work. thank you

**Comment by lllyasviel** [comment]
well it seems that the GPT has some room for improvement,( like only one word prompt)
I will try to improve it next time

**Reply by lllyasviel** [reply]
Edit: All related problems are solved in latest version

**Comment by Imadbein46** [comment]
Play around with PyraCanny
![HDR](https://github.com/lllyasviel/Fooocus/assets/56880122/da32ca87-5195-40d4-ae2c-6e03385f58ea)
![HH](https://github.com/lllyasviel/Fooocus/assets/56880122/492adb8e-3b25-4641-9bd5-cc57dabe0c29)

**Reply by vicento** [reply]
Love Fooocus and Juggernaut XL v6
Use this setting to get great results :
Guidance Scale should be in 3 - 5 range and use no refiner

```
Styles: ['Fooocus V2', 'Fooocus Sharp', 'Fooocus Enhance', 'Fooocus Masterpiece', 'Fooocus Photograph', 'SAI Photographic'], Performance: Quality
Resolution: (..., ...), Sharpness: 4
Guidance Scale: 3, ADM Guidance: (1.5, 0.8)
Base Model: juggernautXL_version6Rundiffusion.safetensors, Refiner Model: None
Sampler: dpmpp_2m_sde_gpu, Scheduler: karras
Seed:, LoRA [sd_xl_offset_example-lora_1.0.safetensors] weight: 0.5
LoRA [xl_more_art-full_v1.safetensors] weight: 0.5, LoRA [add-detail-xl.safetensors] weight: 0.7
```

**Comment by Lennyblonde** [comment]
Hey there everybody,
Is it just me?  Or are there any other people that also experience that with using the sd_xl_base model and with the Realistic Vision V5.1 as a refiner with The refiner switch to 0.4 
illyasviel article: https://github.com/lllyasviel/Fooocus/discussions/802
That you get much better results and less Problems with disformed eyes and other artefacts on the face. 

And I'm also testing a lot now  how this method compares to the run_realistic.bat model.
And it's hard for me to tell which one is better,  maybe it is not about better but more about different methods which you can use to get different results?
 anyway I would be delighted to hear  what other people have experienced with this
 thanks in front
 greetings Lenny blonde

**Comment by rinilkunhiraman** [comment]
**m1 pro 16GB**

```
[System ARGV] ['entry_with_update.py', '--attention-split']
Python 3.10.13 (main, Sep 11 2023, 08:16:02) [Clang 14.0.6 ]
Fooocus version: 2.1.862
Running on local URL:  http://127.0.0.1:7865

To create a public link, set `share=True` in `launch()`.
Total VRAM 16384 MB, total RAM 16384 MB
Set vram state to: SHARED
Always offload VRAM
Device: mps
VAE dtype: torch.float32
Using split optimization for cross attention
Refiner unloaded.
model_type EPS
UNet ADM Dimension 2816
Using split attention in VAE
Working with z of shape (1, 4, 32, 32) = 4096 dimensions.
Using split attention in VAE
```

- Python 3.10 memory usage is around 25GB
- The each images take an average 45 mins to completion.
- This is the same amount of time it takes without --attention-split as well.

 **Is this normal for this M1 hardware?**

**Comment by poor7** [comment]
Interesting test 160! models at high resolution 1024x1024. High resolution testing starts at 33:30.
https://youtu.be/G-oZn4H-aHQ?si=viA9Fddqo_xlyPlA&t=2009

**Comment by Parteim** [comment]
Hello! Can anyone help me? I'm having trouble running Fooocus on an AMD GPU. After launching, in the terminal I see that Fooocus uses 1024 MB of video memory, but my computer has a GPU RX 6600 XT with 8 GB of video memory.
![21312321312312](https://github.com/lllyasviel/Fooocus/assets/48876969/a0cfa83e-8ac7-4c51-a803-d930b4de7eda)
![photo_2024-02-06_09-50-03](https://github.com/lllyasviel/Fooocus/assets/48876969/a7d50752-ac17-48dc-885b-8a0ccd5b777b)
